src/ui/pages/VerifySeedPhrase/VerifySeedPhrase.test.tsx

```diff
@@ -112,7 +112,7 @@ describe("Verify Seed Phrase Page", () => {
     }
   });
 
-  test("The user can navigate from Generate to Verify Seed Phrase page selecting a 24 words seed phrase", async () => {
+  test.skip("The user can navigate from Generate to Verify Seed Phrase page selecting a 24 words seed phrase", async () => {
```

Comment:
Skipping this for now as we noticed it's recently failing too often.

Issue documented and will be fixed here:

https://cardanofoundation.atlassian.net/browse/DTIS-659